app/helpers/layout_helper.rb

```diff
@@ -486,7 +487,7 @@ def last_days(days)
 
   def fullscreen_button(element = "$(this).prev()")
     button_tag(:type => 'button', :class => 'btn btn-default btn-md btn-fullscreen', :onclick => "set_fullscreen(#{element})", :title => _("Full screen")) do
-      icon_text('resize-full')
+      icon_text('expand', '', :kind => 'fa')
```

Comment:
I didn't find the same icon we had for full screen in pficon so I used this one.